engine/src/main/java/org/terasology/engine/TerasologyEngine.java

```diff
@@ -428,34 +429,34 @@ private void mainLoop() {
 
             Iterator<Float> updateCycles = time.tick();
 
-            PerformanceMonitor.startActivity("Network Update");
-            networkSystem.update();
-            PerformanceMonitor.endActivity();
+            try (Activity ignored = PerformanceMonitor.startActivity("Network Update")) {
```

Comment:
Alas, no. The syntax for a try-with-resources block requires the resource to be stored in a variable. ```ignored``` is the convention for resources that aren't used in the block (doesn't prompt a warning from IDEs).